vnext/ReactUWP/EndPoints/dll/react-native-uwp.x86.def

```diff
@@ -1,60 +1,77 @@
+;READFIRST
+;Please keep the ABI surface in sync across all platforms. If you need to add a
+;new EXPORT, add it to the other def files too. The EXPORTs are sorted to aid
+;in comparisons.
```

Comment:
The Win32 export definitions have a similar comment, encouraging folks to contact the CXE team for API review first. I think we should generalize that to a RN4W "API review group". We also have CODEOWNERS set up to help keep an eye on this. 

 If the "review group" suggestions sounds reasonable, what's the best github construct to represent this?

Reply:
I can add another line to the CODEOWNERS as part of this PR. There's already one adding you to for the Desktop *.def files.

Reply:
I've updated the CODEOWNERS to add @microsoft/react-native-windows-api-review to reviews that change def/idl files. We should invite the right people to that team.